### DISCUSSION TITLE
2009/09/11/%E5%A6%82%E4%BD%95%E8%B4%AD%E4%B9%B0%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#277] [Announcements]
# 2009/09/11/%E5%A6%82%E4%BD%95%E8%B4%AD%E4%B9%B0%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B

大的新华书店应该有售。这里列出的主要是网上购书途径，可以选择下面链接之一进行购买：

https://blog.qizhen.xyz/2009/09/11/%E5%A6%82%E4%BD%95%E8%B4%AD%E4%B9%B0%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B

**Comment by ruanqizhen** [comment]
之前博客下的留言：

駐
2011年01月30日 16:54 编辑
您的书看过了,很受用
不知道何时能出PK版本,将继续支持!

回复
Qizhen
2009年09月22日 21:53 编辑
应该不是盗版，我拿到书后也发现有一些图不够清晰。应当是从word文档到出版社专用的排版软件文档转换时产生的损失。我会把这些不清晰的图片贴到博客上来，然后在勘误列表中列出。不过我目前看到的绝大多数图还是清晰的。如果你发现哪一张不清楚，可以立即通知我，我好尽快更新。

回复
jiong
2009年09月22日 16:49 编辑
我在当当买的书怎么感觉是盗版的 书上的图有很多都不清楚

回复
